run-be-ut.sh

```diff
@@ -114,6 +117,7 @@ while true; do
         --help) HELP=1 ; shift ;;
         --with-aws) WITH_AWS=ON; shift ;;
         --with-gcov) WITH_GCOV=ON; shift ;;
+        --without-group) WITHOUT_GROUP=$2; shift 2;;
```

Comment:
excluding_test_suit or we can make it including_test_suit
We can exclude or not include the slow one or huge memory by default

Reply:
Fixed